providers/keap/metadata_test.go

```diff
@@ -28,9 +33,13 @@ func TestListObjectMetadataV1(t *testing.T) { // nolint:funlen,gocognit,cyclop
 			ExpectedErrs: []error{staticschema.ErrObjectNotFound},
 		},
 		{
-			Name:   "Successfully describe multiple objects with metadata",
-			Input:  []string{"campaigns", "products"},
-			Server: mockserver.Dummy(),
+			Name:  "Successfully describe multiple objects with metadata",
```

Comment:
Can you please add a failure case? Ideally even if we fail to do this, we'd want to show the default schema list